### DISCUSSION TITLE
Performance of non-english TTS models

### DISCUSSION THREAD

**Discussion by ivan94fi** [coqui-ai/TTS#1575] [General Q&A]
Hi, first of all the obligatory thank you to the maintainers, this is really a wonderful project.

Then to the question. I am trying to use the italian models to generate some audio files for navigator directions, but the output is really low in quality, almost unusable I would say. Is this expected or am I missing something?

More specifically I tried both vits and glow-tts italian female models, with the default vocoder and with the universal/libri-tts/wavegrad vocoder, but the results are quite bad for every combination.

Just for testing I tried using the default model and vocoder to synthesize some english text and it worked really well, so my conclusions are that the italian model are not particularly good.

**Comment by erogol** [comment]
It is appropriate to ping @[nicolalandro](https://github.com/nicolalandro), who gracefully trained those models. :)
Maybe he has some updates to share.

**Reply by ivan94fi** [reply]
Yes any help would be appreciated. Thank you.
